i18n-util/src/LocaleUtil.js

```diff
@@ -0,0 +1,43 @@
+const LOCALE_COOKIE_KEY = 'okta_user_lang';
```

Comment:
We should discuss this - as this is the same key that `okta-ui-react` uses for locale persistence.

If a user navigates to a help link with `?locale=${someLocale}`, would we expect the text in Enduser/Admin to change the next time they visit?

Reply:
( FYI I am yet to test this out  end to end because cookies don't work with files on your local system). Yes, this is the same key that we set. I believe that the cookie gets set against a domain and here it would be set against help.okta.com domain which would be different than the one on the dashboard app

Reply:
To avoid any potential confusion with the cookie set on the enduser side, going to update this to `okta_help_user_lang`